config/manager/manager.yaml

```diff
@@ -47,10 +47,10 @@ spec:
             periodSeconds: 10
           resources:
             limits:
-              cpu: "600m"
-              memory: "1Gi"
+              cpu: "200m"
+              memory: "80Mi"
```

Comment:
When I look at this service in GCP, it's regular usage sits at ~96MB.
I think shipping this as-is will result in an OOM kill loop.

---

One way to get an idea of this number is to visualize the `max` memory usage over say a month -- Rather than the average. It's common for services to spike in memory temporarily at startup or when processing a lot of data -- And you want some headroom for that.

The limits (if I remember correctly) don't have an effect on bin-packing -- Just killing when things get out of hand.

---

Another way to think about things...

1. Setting `requests` seems like it ought to be generally quite safe for us, it's unlikely that this will impact cluster stability as long as we pick generally appropriate numbers
2. Setting `limits` is less safe
    1. On the CPU side, limits will throttle a process -- Not a huge deal -- We can probably be reasonably tight with CPU limits and that helps avoid "noisy neighbors"
    2. On the memory side, memory _is not_ compressible -- Anytime we hit a limit the process will be killed

For those reasons, I'd say we might try and be "pretty tight" on requests but start pretty loose on limits.